.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+<!--- PR title format: [GH#<Issue ID>][BZ<BZ ID>][<Jira ID>]: <Short description of your PR> --->
+
+
+Version:
+<!--- What version of OpenShift does this apply to? --->
+
+
+Issue:
+<!--- Add a link to your Bugzilla, Jira, or GitHub issue. --->
+
+
+Docs Preview:
+<!--- Add the direct link to your content. --->
+
+Summary of changes
+*
+
+
+Review checklist
+- [ ] SME review
+- [ ] Peer review
+- [ ] QE review
+<!--- The checklist is optional and included for convenience.
+Make a note if your issue does not require certain validation or review steps.
+If you have further questions, consult the OpenShift Docs Manual. --->
```

Comment:
The docs manual is internal, just wondering whether it's appropriate to include here. Maybe it could be to ping a team member or something instead? Or the repo guidelines?